Core/System/Repository/ProductRepository.cs

```diff
@@ -44,6 +56,7 @@ public Product FetchProductData(int productId)
                     product.Category = new Category() { Id = Convert.ToInt32(row["category"]) };
                     product.MetricUnit = new MetricUnit() { Id = Convert.ToInt32(row["metricUnit"]) };
                     product.MetricValue = row["metricValue"].ToString();
+                    product.Status = Convert.ToInt32(row["status"].ToString());
```

Comment:
Is this part of the design? we are fetching the `status` value from the backend, but it will not appear in the frontend.